src/backend/mod.rs

```diff
@@ -23,7 +23,7 @@ pub type Pixel = u16;
 #[derive(Debug, Clone)]
 pub enum MonitorInfo {
     /// Some additional values in the X11 context.
-    X11 { name: u32 },
+    X11 { output_name: String },
```

Comment:
Can we name it just `name`? It would be equal to the Wayland counterpart then.

Reply:
ok

Reply:
done